### PR TITLE
Updated the `origin-local-checkout` helptext to be more correct.

### PR DESCRIPTION
**Pull request by stevekuznetsov**

### CODE REVIEW

File: lib/vagrant-openshift/command/checkout_repositories.rb [hunk 1]
```diff
@@ -33,7 +33,7 @@ def execute
             o.banner = "Usage: vagrant checkout-repos"
             o.separator ""
 
-            o.on("-b [branch-name]", "--branch [branch-name]", String, "Check out the specified branch. Default is 'master'.") do |f|
+            o.on("-b [branch-name]", "--branch [branch-name]", String, "Check out the specified branch. If unset, no checkout happens and the default branch is used..") do |f|
```
Comment (danmcp) [inline lib/vagrant-openshift/command/checkout_repositories.rb]: why end in two dots?

How about end with (i.e. master).

**Comment by stevekuznetsov** [comment]
@danmcp addressed comment

File: lib/vagrant-openshift/command/checkout_repositories.rb [hunk 2]
```diff
@@ -33,8 +33,7 @@ def execute
             o.banner = "Usage: vagrant checkout-repos"
             o.separator ""
 
-            o.on("-b [branch-name]", "--branch [branch-name]", String, "Check out the specified branch. Default is 'master'.") do |f|
-              options[:branch] = {"origin-server" => f}
+            o.on("-b [branch-name]", "--branch [branch-name]", String, "Check out the specified branch. If unset, no checkout happens and the default branch is used (e.g. master).") do |f|              options[:branch] = {"origin-server" => f}
```
Comment (danmcp) [inline lib/vagrant-openshift/command/checkout_repositories.rb]: why move the assignment to the same line?

Comment (stevekuznetsov) [inline lib/vagrant-openshift/command/checkout_repositories.rb]: sloppy, fat fingers. sorry

**Pull request merged**